tests/fsharpqa/Source/Conformance/SpecialAttributesAndTypes/Imported/CallerInfo/CallerLineNumber.fs

```diff
@@ -0,0 +1,41 @@
+namespace Test
+
+open System.Runtime.CompilerServices
+open CSharpLib
+[<MyCallerInfo>]
+type MyTy() =
+    static member GetCallerLineNumber([<CallerLineNumber>] ?line : int) =
```

Comment:
Add tests for first-class uses of ``GetCallerLineNumber`` and friends (and record the resolution of any design issues that arise in the RFC)